src/EmbeddedVsix/Manifests/VS2022/source.extension.vsixmanifest

```diff
@@ -11,12 +11,12 @@
         </InstallationTarget>
     </Installation>
     <Dependencies>
-        <Dependency Id="Microsoft.Framework.NDP" DisplayName="Microsoft .NET Framework" d:Source="Manual" Version="[4.5,)" />
+        <Dependency Id="Microsoft.Framework.NDP" DisplayName="Microsoft .NET Framework" d:Source="Manual" Version="[4.7.2,)" />
```

Comment:
Minor tweaks to the VS2022 manifest to make it match the conditions in Integration.VSIX.